src/internal/list.js

```diff
@@ -0,0 +1,2 @@
+export const empty = ({isEmpty: true, size: 0, head: null, tail: null});
+export const cons = (head, tail) => ({isEmpty: false, size: tail.size + 1, head, tail});
```

Comment:
I did include them for aesthetic reasons. :)
I'll keep it as empty, as I consider it more descriptive within the context of a list. I renamed it `emptyList` during import to `core.js`, to preserve context.